text_generation/causal_lm/cpp/continuous_batching/python/tests/common.py

```diff
@@ -91,6 +91,7 @@ def get_scheduler_config(scheduler_params: dict = None) -> SchedulerConfig:
         # vLLM specific
         scheduler_config.max_num_batched_tokens = 256
         scheduler_config.max_num_seqs = 256
+        scheduler_config.cache_size = 2
```

Comment:
this can be not enough for real models where hidden_size can be very big.

Reply:
or maybe we can specify size in blocks? We usually generate 4 sequence group with max_new_tokens = 30, so we can roughly estimate a number of required blocks.

Reply:
Changed to setting of `num_kv_blocks` instead of `cache_size` and added comment with approximate estimation of required blocks. Please take a look.